docs/src/app/experiments/dialog.module.css

```diff
@@ -188,22 +188,22 @@
   padding: 16px;
   font-family: IBM Plex Sans;
 
-  h1 {
+  & h1 {
```

Comment:
All of a sudden CSS Modules started acting out, saying the selector isn't pure? Not sure how these are connected. Anyway, I like explicit `&`